ctrd/image_proxy_util.go

```diff
@@ -66,7 +66,7 @@ func canonicalAddr(url *url.URL) string {
 // change no_proxy to noProxy for avoid fail in go lint check
 var (
 	noProxyEnv = &envOnce{
-		names: []string{"NO_PROXY", "noProxy"},
+		names: []string{"NO_PROXY", "no_proxy"},
```

Comment:
Could you help to confirm this? Since I think it is introduced by your part. @Ace-Tang

Reply:
have communicated with @Ace-Tang offline. The change is totally reasonable.